### PR TITLE
Added my project 'FINADICT' to the list

### PR DESCRIPTION
**Pull request by subhayu99**
Added all the details of the project as per the requirements.

### CODE REVIEW

File: projects/subhayu99/finadict.mdx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  - Different types of plots for the data.
+  - Option to run locally or on a server using docker.
+
+</Overview>
+
+<Contributing>
+
+**Contributing FAQs**
+
+  - What kind of contributions are you looking for?
```
Comment (ramonh) [inline projects/subhayu99/finadict.mdx:60]: Thanks so much for your contribution!

I'm afraid the Markdown came out a little odd in action. Please see this screenshot:
image

Could you please take a look? Thanks in advance!

**Review by ramonh — CHANGES_REQUESTED** [review]
Looking great, love the Map! To the point :)

Please see my feedback below. Thanks!

File: projects/subhayu99/finadict.mdx (hunk 1 above)
Comment (subhayu99) [reply projects/subhayu99/finadict.mdx:60]: Thank you for letting me know.
Actually, I edited the file on vim and forgot to check the actual output.😅
Will change it immediately.

**Comment by subhayu99** [comment]
@ramonh I have ammended the changes you requested and added a few more information on the page.

File: projects/subhayu99/finadict.mdx (hunk 1 above)
Comment (ramonh) [reply projects/subhayu99/finadict.mdx]: Woopwoop! Fellow vim user 😄 

Cool, thanks for fixing!

File: projects/subhayu99/finadict.mdx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+repoUrl: https://github.com/subhayu99/finadict
+description: A simple and easy to use WebApp for Stocks / Cryptocurreny / Foreign Exchange prices prediction.
+languages: # A list of programming languages used in this project
+  - Python
+tags: # A list of keywords relevant to the contributors for this project
+  - First Timer Friendly
+  - Machine Learning
+  - Data Visualization
+  - Web Application
+  - Finance Prediction
```
Comment (ramonh) [inline projects/subhayu99/finadict.mdx:12]:
```suggestion
  - Finance
```

Hope it's fine if I shorten this to `Finance` so it's more reusable!

**Review by ramonh — APPROVED** [review]

**Comment by ramonh** [comment]
Looks wonderful, thank you! Welcome to Port! 🚢

**Comment by subhayu99** [comment]
Thanks for approving ❤️
